src/SlippiGame.ts

```diff
@@ -181,6 +187,12 @@ export class SlippiGame {
       actionCounts: this.actionsComputer.fetch(),
       overall: overall,
       gameComplete,
+      targetBreaks: settings.gameMode === GameMode.TARGET_TEST ? this.targetBreakComputer.fetch() : null,
+      // homerun distance depends on the language setting (not NTSC/PAL) and JPN has not been implemented
+      homerunDistance:
+        settings.gameMode === GameMode.HOME_RUN_CONTEST && settings.language !== Language.JAPANESE
+          ? this.homerunComputer.fetch()
+          : null,
```

Comment:
I don't really think these belong in the `getStats()` call or the `StatsType` object. They're very unrelated to the rest of the stats which mainly only make sense in a competitive environment. I would prefer like a separate call `getStadiumStats()` which could return a `StadiumStats` type instead.